Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -455,6 +466,22 @@
     m_scrollContainerLayer->setEventRegion(WTFMove(eventRegion));
 }
 
+bool UnifiedPDFPlugin::requestScrollToPosition(const ScrollPosition& position, const ScrollPositionChangeOptions& options)
```

Comment:
Should the other code (zooming) that requests a scroll make use of this instead of going straight to scrollingcoordinator?

Reply:
Probably. Will go find it (we have a bug about zooming).

Reply:
setPageScaleFactor just above